src/obu.rs

```diff
@@ -549,6 +541,47 @@ fn parse_seq_hdr(c: &mut Rav1dContext, gb: &mut GetBits) -> Rav1dResult<Rav1dSeq
     })
 }
 
+pub(crate) unsafe fn rav1d_parse_sequence_header(
+    mut data: &[u8],
+) -> Rav1dResult<DRav1d<Rav1dSequenceHeader, Dav1dSequenceHeader>> {
+    while !data.is_empty() {
+        let mut gb = GetBits::new(data);
+
+        gb.get_bit(); // obu_forbidden_bit
+        let raw_type = gb.get_bits(4);
+        let r#type = Rav1dObuType::from_repr(raw_type as usize);
```

Comment:
```suggestion
        let r#type = Rav1dObuType::from_repr(gb.get_bits(4) as usize).unwrap();
```

Reply:
Not sure that's the right thing to do. Not all 16 possible values of `Rav1dObuType` are defined. As per the AV1 specification, values that aren't recognized should simply be ignored. We shouldn't panic when encountering one of the undefined values.

Reply:
Oh okay, I might've misread that.